shared/actions/config/index.js

```diff
@@ -33,6 +34,79 @@ function getConfig (): AsyncAction {
   }
 }
 
+let followers = {}
```

Comment:
the thing that'll be messy is that i'll have to plumb through getState to all these promises since its functions calling functions calling functions etc